scripts/setup_web1.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# CAUTION: To git clone olsystem, environment variables must be set...
+# Set $GITHUB_USERNAME or $USER will be used.
+# Set $GITHUB_TOKEN or this script will halt.
+if [[ -z ${GITHUB_TOKEN} ]]; then
+    echo "FATAL: Can not git clone olsystem" ;
+    exit 1 ;
+fi
+
+# apt list --installed
+sudo apt-get update
+sudo apt-get install -y docker.io docker-compose
+docker --version        # 19.03.8
+docker-compose version  #  1.25.0
+sudo systemctl start docker
+sudo systemctl enable docker
+
+sudo groupadd --system openlibrary
+sudo useradd --no-log-init --system --gid openlibrary --create-home openlibrary
+
+cd /opt
+ls -l  # nothing
+
+# sudo git clone https://github.com/internetarchive/openlibrary
+sudo git clone https://github.com/cclauss/openlibrary
+sudo git clone https://${GITHUB_USERNAME:-$USER}:${GITHUB_TOKEN}@github.com/internetarchive/olsystem
+# sudo git pull https://${GITHUB_USERNAME:-$USER}:${GITHUB_TOKEN}@github.com/internetarchive/olsystem.git master
+sudo chown openlibrary /opt/*
+ls -l  # containerd, olsystem, openlibrary owned by openlibrary
```

Comment:
What's `containerd`?

Reply:
tl;dr An essential building block for Docker and K8
* https://www.docker.com/blog/what-is-containerd-runtime/
* https://containerd.io/

Reply:
Ahh; how did it get here? Did you have to check it out, or is it something that gets added automatically somehow?

Reply:
https://github.com/internetarchive/openlibrary/pull/3938/files#diff-3b30bef8970e22874431ef6aa3d67073bddc40ba78dc7e5a6c86e549a15b5706R13